src/site/content/en/metrics/inp/index.md

```diff
@@ -152,6 +223,10 @@ It's possible that a page can return no INP value. This can happen for a number
 - The page was loaded, but the user interacted with the page using gestures that did not involve clicking, tapping, or using the keyboard. For example, scrolling or hovering over elements does not factor into how INP is calculated.
 - The page is being accessed by a bot such as a search crawler or headless browser that has not been scripted to interact with the page.
 
+## What if an interaction causes no layout changes?
+
+If you have an interaction with an event callback that causes no change to the layout in an event handler, that interaction can still be a candidate for INP. In these cases, you should be thinking about what sort of visual feedback you _could_ provide for the user that lets them know that something is happening as a result of the interaction. The intent of INP is to measure the delay before any potential paint can complete, and that involves recognizing _when_ you have an opportunity to present the next frame to the user. In the case of asynchronous work especially, that frame should communicate that work is in progress as soon as possible.
```

Comment:
This paragraph talks a lot about event callbacks that don't do anything, but the much more common case is that there aren't any event callbacks at all.

For example, if I tap on the body of a page, that will cause the browser to dispatch a number of events which the page may or may not have registered listeners for. But if the main thread is busy and takes a while to respond, that interaction could still become the INP candidate even if there were not event listeners registered for those particular events.

See the FID doc for how this is discussed there: https://web.dev/fid/#what-if-an-interaction-doesnt-have-an-event-listener